src/pages/tasks/TaskDescriptionPage.js

```diff
@@ -48,6 +48,17 @@ function TaskDescriptionPage(props) {
 
     const inputRef = useRef(null);
 
+    // Same as NewtaskDescriptionPage, use the selection to place the cursor correctly if there is prior text
+    const [selection, setSelection] = useState({start: 0, end: 0});
+
+    // eslint-disable-next-line rulesdir/prefer-early-return
+    useEffect(() => {
+        if (props.task.report && props.task.report.description) {
+            const length = props.task.report.description.length;
+            setSelection({start: length, end: length});
+        }
+    }, [props.task.report]);
```

Comment:
Should have been a custom hook maybe. We try to follow the DRY principle.

Reply:
Just saw these comments, sorry. Yeah, this would probably work but I'll be removing it anyway due to the React Native iOS bug with the selection prop.